backend/controller/controller.go

```diff
@@ -490,6 +489,33 @@ func (s *Service) PullSchema(ctx context.Context, req *connect.Request[ftlv1.Pul
 	})
 }
 
+func (s *Service) UpdateModuleRuntime(ctx context.Context, req *connect.Request[ftlv1.UpdateModuleRuntimeRequest]) (*connect.Response[ftlv1.UpdateModuleRuntimeResponse], error) {
+	schemas, err := s.dal.GetActiveDeploymentSchemasByDeploymentKey(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("could not get schemas: %w", err)
+	}
+	for deployment, module := range schemas {
+		if module.Name != req.Msg.Module {
```

Comment:
This should use the deployment key at some point, but we don't have it in the schema runtime data yet.